### PR TITLE
Add public WebSocket APIs to index page - Closes #1277

### LINKED ISSUE #1277: Introduce public Lisk Service APIs
### Description

- service.lisk.io
- mainnet-service.lisk.io
- service.lisk.com
- mainnet-service.lisk.com

- `wss://service.lisk.com/rpc-v2`
- `wss://service.lisk.com/blockchain`

- https://lisk.com/documentation/lisk-service/references/rpc-api.html

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1277

### CODE REVIEW

**Review by chris529 — DISMISSED** [review]
LGTM! 👍

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -149,22 +152,33 @@ curl -X GET "http://localhost:9901/api/v2/forgers" -H  "accept: application/json
...
+node --version
+# v12.22.7
+npm i socket.io-client@2 #<1>
+npm i jsome #<2>
 ----
 
+<1> Use the {url_npm_socketio_client}[socket.io-client^] to connect to the RPC API.
+<2> Optionally install `jsome` to prettify the API response.
+
 IMPORTANT: Specify version 2 of the socket.io client, as the current version 3 is not compatible with Lisk Service.
```
Comment (MichalTuleja) [inline modules/ROOT/pages/index.adoc]: It's not important anymore, Lisk Service already migrated to socket.io version 4.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -208,14 +240,20 @@ npm i socket.io-client@2
 
 IMPORTANT: Specify version 2 of the socket.io client, as the current version 3 is not compatible with Lisk Service.
```
Comment (MichalTuleja) [inline modules/ROOT/pages/index.adoc]: Same as above

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -9,6 +9,10 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 :url_api_mainnet: https://service.lisk.com/api/
 :url_api_testnet: https://testnet-service.lisk.com/api/
+:url_rpc_api_mainnet: wss://service.lisk.com/rpc-v2
+:url_rpc_api_testnet: wss://testnet-service.lisk.com/rpc-v2
+:url_subscribe_api_mainnet: wss://testnet-service.lisk.com/blockchain
+:url_subscribe_api_testnet: wss://testnet-service.lisk.com/blocckhain
```
Comment (MichalTuleja) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:url_subscribe_api_testnet: wss://testnet-service.lisk.com/blockchain
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -255,3 +293,10 @@ subscribe('update.fee_estimates');
 	});
```
Comment (MichalTuleja) [inline modules/ROOT/pages/index.adoc:289]: `Received data from ${WS_SUBSCRIBE_ENDPOINT}/${eventName}: ${newData}`

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Looks good, with some minor remarks and typos.

**Review by MichalTuleja — APPROVED** [review]
👍

**Pull request merged**